v-spec.adoc

```diff
@@ -2624,11 +2624,16 @@ masked va >= x, vd != v0
   pseudoinstruction: vmsge{u}.vx vd, va, x, v0.t
   expansion: vmslt{u}.vx vd, va, x, v0.t; vmxor.mm vd, vd, v0
 
-masked va >= x, any vd
+masked va >= x, vd == v0
 
   pseudoinstruction: vmsge{u}.vx vd, va, x, v0.t, vt
   expansion: vmslt{u}.vx vt, va, x;  vmandnot.mm vd, vd, vt
 
+masked va >= x, any vd
+
+  pseudoinstruction: vmsge{u}.vx vd, va, x, v0.t, vt
+  expansion: vmslt{u}.vx vt, va, x;  vmandnot.mm vt, v0, vt;  vmandnot.mm vd, vd, v0;  vmor.mm vd, vt, vd
```

Comment:
This is so complicated! At some point it _must_ preferable just to splat `x` to `vt` and use a `.vv` version. (@aswaterman @kasanovic?)